app/models/division_team.rb

```diff
@@ -0,0 +1,6 @@
+class DivisionTeam < ApplicationRecord
+
+  belongs_to :division
```

Comment:
Assuming this join table is so we can "reuse" teams in future divisions, yes? Couldn't think of any other join tables we would need to make the magic happen.

Reply:
I'm thinking we also want to keep the history in place. That way, it's easy to navigate back to previous seasons and check out divs/teams/stats/etc.